.gitignore

```diff
@@ -1,3 +1,4 @@
 .phpunit.cache
 composer.lock
 vendor
+.idea
```

Comment:
This should be a global setting, you can check on [this docs](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) in the `Configuring ignored files for all repositories on your computer` section in order to do it globally 👍